### ISSUE TITLE
[1.18.1] Crash on new world screen in 2.3.0 after modifying config file.

### ISSUE THREAD

**Issue by AnotherVagabond** [FalsehoodMC/Fabrication#383]
Whenever I enter single-player and it moves me to the create new world screen, the game crashes out. This doesn't happen with a completely fresh install of Fabrication, only after I modified the configs. The configs were modified using Cloth Config API 6.1.48 and Mod Menu 3.0.1 on a different profile, but the same set of configs caused the same crash when copied over to a fresh install of just Fabric and Fabrication.

I've included the config file that was used, as well as the logs from the fully modded version in case those help.

Fresh Install Log: https://gist.github.com/AnotherVagabond/1f8efd3295d512c3b01d40ed09723a8e

Config: https://gist.github.com/AnotherVagabond/6333db2194b95ea4576195b036658790
Fully Modded Log: https://gist.github.com/AnotherVagabond/6b4c91af6199b62d5bd9fb68ff3a922e

**Comment by exaskye** [comment]
Disable Encroaching Emeralds. Duplicate of #373

**Comment by exaskye** [comment]
I need to release the fix for this but haven't gotten around to it yet.

**Comment by AnotherVagabond** [comment]
That did it, thanks!
